configuration.yaml

```diff
@@ -0,0 +1,490 @@
+        XLA computation, and chain-executed). This environment variable, if set
+        to true, enables OpByOp during the "get tensors" operation (the
+        operation used by PyTorch/XLA to fetch intermediate values back from
+        the TPU device into PyTorch CPU tensors).
+    type: bool
+    default_value: false
+  XLA_SYNC_TENSORS_OPBYOP:
+    description:
+      - The same as XLA_GET_TENSORS_OPBYOP but for "sync tensors" operation(the
+        operation used at the end of a step, to flush pending IR computations
+        and materialize them into TPU device data).
+    type: bool
+    default_value: false
+  XLA_SYNC_WAIT:
+    description:
+      - Forces the XLA tensor sync operation to wait for its completion, before
+        moving to the next step.
+    type: bool
+    default_value: false
+  TF_CPP_LOG_THREAD_ID:
+    description:
+      - If set to true, the TF logs will show the thread ID helping with debugging
+        multithreaded processes.
+    type: bool
+    default_value: false
+  XLA_DUMP_HLO_GRAPH:
+    description:
+      - If set to true in case of a compilation or execution error the
+        offending HLO graph will be dumped as part of the runtime error raised
+        by xla_util.cc.
+    type: bool
+    default_value: false
+  XLA_TEST_DUMP_METRICS:
+    description:
+      - Controls whether or not metrics are dumped in test tear down.
+    type: bool
+    default_value: false
+  XLA_NO_SPECIAL_SCALARS:
+    description:
+      - When set to false, this will enable a compiler optimization to route
+        some tensor values to constant scalars.
+    type: bool
+    default_value: false
+  XLA_TEST_DUMP_TENSORS:
+    description:
+      - Whether or not to print tensors to std::cerr in CPP tests.
+    type: bool
+    default_value: false
+  XLA_TENSOR_UPDATE_SYNC:
+    description:
+      - Used to decide whether or not to sync update in
+        XLANativeFunctions::_copy_from.
+    type: bool
+    default_value: true
+  XLA_DUMP_FATAL_STACK:
+    description:
+      - Installs the stack trace handler upon XLA client creation.
+    type: bool
+    default_value: false
+  VERSIONED_XLA_BUILD:
+    description:
+      - Creates a versioned build. In particular, appends a git sha to the
+        version number string
+    type: bool
+    default_value: false
+  COMPILE_PARALLEL:
+    description:
+      - Enabled parallel compile for PyTorch/XLA build.
+    type: bool
+    default_value: true
+  BUILD_CPP_TESTS:
+    description:
+      - Whether or not to build the cpp tests.
+    type: bool
+    default_value: true
+  XLA_DEBUG:
+    description:
+      - Whether or not to build xla and the xrt client in debug mode.
+    type: bool
+    default_value: false
+  XLA_BAZEL_VERBOSE:
+    description:
+      - Turn on verbose messages during the bazel build of the xla/xrt client.
+    type: bool
+    default_value: false
+  XLA_CUDA:
+    description:
+      - Build the xla/xrt client with CUDA enabled.
```

Comment:
Change to `XLA client` since this applies to both XRT and PJRT

Reply:
Done